### ISSUE TITLE
AgileObjects.ReadableExpressions, Version=4.1.4.0 ???

### ISSUE THREAD

**Issue by ziggy90127** [agileobjects/eg-automapper-readableexpressions#1]
In the .net core 3.1 project I cloned from here, I get the error below.  I built a new .net framework 4.6.1 project and it works fine.  As of this writing, AgileObjects.ReadableExpressions Version=4.1.3 is the latest version, don't know why it's asking for 4.1.4.

`System.IO.FileLoadException: Could not load file or assembly 'AgileObjects.ReadableExpressions, Version=4.1.4.0, Culture=neutral, PublicKeyToken=9f54ad81db69da8e'. 
Could not find or load a specific file. (0x80131621)

File name: 'AgileObjects.ReadableExpressions, Version=4.1.4.0, Culture=neutral, PublicKeyToken=9f54ad81db69da8e'
---> System.IO.FileLoadException: Could not load file or assembly 'AgileObjects.ReadableExpressions, Version=4.1.4.0, Culture=neutral, PublicKeyToken=9f54ad81db69da8e'.
   at System.Runtime.Loader.AssemblyLoadContext.LoadFromPath(IntPtr ptrNativeAssemblyLoadContext, String ilPath, String niPath, ObjectHandleOnStack retAssembly)
   at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyPath(String assemblyPath)
   at System.Reflection.Assembly.LoadFrom(String assemblyFile)
   at System.Reflection.Assembly.LoadFromResolveHandler(Object sender, ResolveEventArgs args)
   at System.Runtime.Loader.AssemblyLoadContext.InvokeResolveEvent(ResolveEventHandler eventHandler, RuntimeAssembly assembly, String name)
   at System.Runtime.Loader.AssemblyLoadContext.OnAssemblyResolve(RuntimeAssembly assembly, String assemblyFullName)
   at AgileObjects.ReadableExpressions.Visualizers.ObjectSource.ExpressionVisualizerObjectSource.GetTranslationFor(Object target)
   at AgileObjects.ReadableExpressions.Visualizers.ObjectSource.ExpressionVisualizerObjectSource.GetData(Object target, Stream outgoingData, Action2 serializer)`

**Comment by SteveWilkes** [comment]
Thanks for letting me know about this -  looks like v4.7.0 of the Visualizers had a bug - I've now released v4.7.1, which is working as expected:

![AutoMapperReadableExpressionsDemo](https://github.com/user-attachments/assets/abe93cab-1484-4519-a6ad-da94492cd2a9)

**Comment by ziggy90127** [comment]
Thank you.  After adding 
```
<PropertyGroup>
   <LangVersion>10.0</LangVersion>
</PropertyGroup>
```
To the .csproj file, it works great!
